docs/publications.md

```diff
@@ -9,7 +9,7 @@ permalink: /publications/
 # Publications
 
 [FuzzBench](https://dl.acm.org/doi/pdf/10.1145/3468264.3473932)
-[[BibTeX](https://ieeexplore.ieee.org/abstract/document/9787836)]
+[[BibTeX](https://scholar.googleusercontent.com/scholar.bib?q=info:ELh3M_umuJoJ:scholar.google.com/&output=citation&scisdr=CgXp-uhUEOWDvhWGNzg:AAGBfm0AAAAAY42ALzg1eyEmlFpd8_hvdW7JVBxoPP5m&scisig=AAGBfm0AAAAAY42AL1eb8vNiL_vvBm3HWkO1EEQPHzke&scisf=4&ct=citation&cd=-1&hl=en)]
```

Comment:
This link doesn't seem to work:

403. That’s an error.

Your client does not have permission to get URL /scholar.bib?q=info:ELh3M_umuJoJ:scholar.google.com/&output=citation&scisdr=CgXp-uhUEOWDvhWGNzg:AAGBfm0AAAAAY42ALzg1eyEmlFpd8_hvdW7JVBxoPP5m&scisig=AAGBfm0AAAAAY42AL1eb8vNiL_vvBm3HWkO1EEQPHzke&scisf=4&ct=citation&cd=-1&hl=en from this server.

Reply:
It seems Google scholar changes the` scisdr` and `scisig` fields in the URL after a while and makes the old link unavailable.
Do you happen to know if we have a permanent link for it?
I may have to delete this BibTex otherwise : (

Reply:
Consider using this:

https://google.github.io/fuzzbench/faq/#how-can-i-cite-fuzzbench-in-my-paper

Reply:
Thanks, done : )